homeassistant/config.py

```diff
@@ -244,18 +246,20 @@ def set_time_zone(time_zone_str):
 
     set_customize(config.get(CONF_CUSTOMIZE) or {})
 
-    if CONF_TEMPERATURE_UNIT in config:
-        hac.temperature_unit = config[CONF_TEMPERATURE_UNIT]
+    if CONF_UNIT_SYSTEM in config:
+        hac.units = IMPERIAL_SYSTEM \
```

Comment:
Well one is a string, one is a unit system.

Would you prefer `IMPERIAL_SYSTEM_NAME` or `CONF_UNIT_SYSTEM_IMPERIAL` for the names?